src/include/utils/left_recursion.h

```diff
@@ -0,0 +1,14 @@
+#ifndef JUCC_LEFT_RECURSION_H
+#define JUCC_LEFT_RECURSION_H
+#include "grammar/grammar.h"
+
+namespace jucc::utils {
+const char DASH[] = "'";
+
+grammar::Productions RemoveDirectLeftRecursion(const grammar::Production & /*prod*/);
+bool IsRecursive(const grammar::Production & /*prod*/);
+
+grammar::Productions RemoveIndirectLeftRecursions(const grammar::Productions &);
```

Comment:
Function declaration unnamed parameter.
Not commented header file

Reply:
Just kind of an interface, no implementation required till now.

Reply:
👀 Not commented header file

Reply:
Done!!